### PR TITLE
Updated practice2

### PR DESCRIPTION
**Pull request by KirinaKatya**
I did it

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+idna==3.4
+numpy==1.26.0
+pandas==2.1.1
+python-dateutil==2.8.2
+pytz==2023.3.post1
+requests==2.31.0
+six==1.16.0
+sqlparse==0.4.4
+tzdata==2023.3
+urllib3==2.0.5
```
Comment (esinkirill) [inline requirements.txt:14]: тут круто

File: README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Practice 2
+
+* настройка виртуального окружения
+* добавление библиотек
+* добавление файлов *requirements* и *README*
```
Comment (esinkirill) [inline README.md:5]: md в порядке

File: Main.py [hunk 3]
```diff
@@ -0,0 +1,2 @@
+import requests 
+# а дальше импортировались пакеты через само приложение
```
Comment (esinkirill) [inline Main.py:2]: библия

**Review by esinkirill — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (KirinaKatya) [reply requirements.txt:14]: спасибо, стараемся. r u seriously want to merge it???

**Pull request merged**